mock/validator_test.go

```diff
@@ -30,7 +30,8 @@ func TestValidator(t *testing.T) {
 
 	// setup a simple HTTP handler that just outputs status 204, 401 or 403 depending on auth result
 	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if !v.CheckToken(r).Require(w, "api:access") {
+		if !v.CheckToken(r).Check("api:access") {
```

Comment:
This does not match how `CheckToken` is supposed to work. Indeed, the `mock.Validator` struct does not satisfy the `gophercloud.Validator` interface anymore. This can also be reproduced by adding the following to `mock/validator.go`:

```go
var _ gopherpolicy.Validator = &Validator[*policy.Enforcer]{}
```